bedrock/mozorg/templates/mozorg/cms/about/blocks/leadership_block.html

```diff
@@ -37,7 +37,7 @@ <h3 class="fn" itemprop="name">{{ value.name }}</h3>
 
   {% if value.biography %}
     <div class="person-bio w-rich-text">
-      {{ value.biography|richtext }}
+      {{ value.biography|wagtail_richtext|safe }}
```

Comment:
Swapping this to our custom `wagtail_richtext` filter is breaking tests because it breaks the Factory we use to bootstrap leadership page data in tests (and will also break any other tests we want to use factories for....